### ISSUE TITLE
!trill(! and !trill)! seem to not work

### ISSUE THREAD

**Issue by rualark** [paulrosen/abcjs#288]
I try to use it like this:
```
AG!trill(!BDA!trill)!DFDAD
```
Result:
![image](https://user-images.githubusercontent.com/15198808/71781840-4e3eaa00-2fe4-11ea-807d-d651a76de7de.png)

**Comment by paulrosen** [comment]
Yes - that isn't implemented yet. I will try to get to this soon.
